### PR TITLE
adds the island movie

### PR DESCRIPTION
**Pull request by Ibrahim-Islam**
## Checklist for submitting a pull request to `Movies For Hackers`:

- [ x] I have carefully **read and comply** with the [Contributing Guidelines](https://github.com/k4m4/movies-for-hackers/blob/master/contributing.md) of this repo.
- [ x] I have **searched** the [PRs](https://github.com/k4m4/movies-for-hackers/pulls) of this repo and believe that this is not a duplicate.

 

- **Title**: The Island
- **Link**: https://www.imdb.com/title/tt0399201
- **Genre**: Action/Adventure
- **Rating**: 6.9

<br/ >

- **Description**: A man living in a futuristic sterile colony begins to question his circumscribed existence when his friend is chosen to go to the Island, the last uncontaminated place on earth.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -14,14 +14,16 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
...
 
-1. [Movies For Hackers](#movies-for-hackers)
-    - [Thrillers/Drama](#thrillers--drama)
-    - [Science Fiction/Fantasy](#science-fiction--fantasy)
-    - [Action](#action)
-    - [Documentaries](#documentaries)
-1. [TV Shows For Hackers](#tv-shows)
-1. [Other Cool Lists](#other-cool-lists)
-1. [Contributing](#contributing)
+- [Movies For Hackers ![Awesome](https://github.com/sindresorhus/awesome)](#movies-for-hackers-awesome)
```
Comment (styfle) [inline readme.md]: Why did you change this?

**Comment by k4m4** [comment]
Thanks @styfle & @Ibrahim-Islam!
